src/app/unlocked-stage/unlocked-stage.component.ts

```diff
@@ -41,32 +43,42 @@ export class UnlockedStageComponent implements OnInit {
   private mosquitoes = [];
   private maxMosquitoes = 10;
   private mosquitoesStrength = Array(this.maxMosquitoes).fill(100);
-  private mosCountInitial = 2;
-  private mosCountCurrent = this.mosCountInitial;
+  private mosCountCurrent: number;
   private keydown = false;
   private levelCounter = 1;
-  private maxLevel = 4;
+  private level: number;
+  public readonly CURR_STAGE = 10;
   private readonly HEALTH_REDUCTION = 5;
   private readonly NET_POSITION_CONSTRAINT = {minX: 1200, maxX: 1270, maxY: 517};
   private readonly SPRAY_OFFSET = {x: 510, y: 150};
-  private readonly NET_OFFSET = {y: 150};
+  private readonly NET_offset = {y: 150};
```

Comment:
even this is the same way.. and it will break the code where you are trying to refer to NET_OFFSET.. please verify once.

Reply:
Actually, it was changed there as well where it was called. Anyway, fixed that.